docs/src/components/Sidenav/index.tsx

```diff
@@ -0,0 +1,135 @@
+import * as React from "react";
+import styled, { css } from "styled-components";
+import Menu from "@kiwicom/orbit-components/lib/icons/MenuHamburger";
+import Close from "@kiwicom/orbit-components/lib/icons/Close";
+import mq from "@kiwicom/orbit-components/lib/utils/mediaQuery";
+import useClickOutside from "@kiwicom/orbit-components/lib/hooks/useClickOutside";
+
+interface WrapperProps {
+  width: number;
+  shown: boolean;
+}
+
+const StyledAsideWrapper = styled.aside<WrapperProps>`
+  ${({ width, theme, shown }) => css`
+    background: ${theme.orbit.paletteWhite};
+    padding: 10px 20px;
+    z-index: 100;
+    margin: 0;
+    display: block;
+    position: absolute;
+    box-sizing: border-box;
+    top: 0;
+    bottom: 0;
+    height: 100vh;
+    overflow-y: auto;
+    font-family: ${theme.orbit.fontFamily};
+    box-shadow: ${theme.orbit.boxShadowRaised};
+    visibility: ${shown ? `visible` : `hidden`};
+    transition: transform ${theme.orbit.durationFast} ease-in;
+    width: 100%;
+    right: 0;
+    transform: translate3d(${shown ? "0, 0, 0" : `100%, 0, 0`});
+    ${mq.largeMobile(css`
+      max-width: ${width}px;
+    `)};
+  `}
+`;
+
+const StyledOpenButton = styled(({ className, children, ariaLabel, onClick }) => (
+  <button className={className} aria-label={ariaLabel} onClick={onClick} type="button">
+    {children}
+  </button>
+))`
```

Comment:
I believe you can handle these situations better with [`.attrs`](https://styled-components.com/docs/api#attrs), then it's instantly clear that you're only converting `ariaLabel` to `aria-label` and adding `type="button"`.